geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/coprocessor/GeoMesaCoprocessor.scala

```diff
@@ -66,8 +66,9 @@ class GeoMesaCoprocessor extends GeoMesaCoprocessorService with Coprocessor with
         val scanner = env.getRegion.getScanner(scan)
         aggregator.setScanner(scanner)
         try {
-          while (aggregator.hasNextData) {
-            results.append(aggregator.aggregate())
+          val agg = aggregator.aggregate()
```

Comment:
We removed it because aggregator.aggregate() has a while loop that also calls hasNextData https://github.com/aheyne/geomesa/blob/7bca0fcb49e9a068137bc21414af34a9073b85e1/geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/iterators/AggregatingScan.scala#L87-L87